js/packages/teams-ai/src/TestTurnState.ts

```diff
@@ -0,0 +1,26 @@
+import { TurnState } from "./TurnState";
```

Comment:
Should this file go under internals?

Reply:
I think they're useful for 3rd parties who want to test their bots.